website/docs/cdktf/concepts/resources.mdx

```diff
@@ -381,6 +381,236 @@ If you need to use the special [`self` object](/terraform/language/resources/pro
+                   arn                         = "arn:aws:s3:::best-bucket-in-the-world"
+                   bucket                      = "best-bucket-in-the-world"
+                   bucket_domain_name          = "best-bucket-in-the-world.s3.amazonaws.com"
+                   bucket_regional_domain_name = "best-bucket-in-the-world.s3.us-east-1.amazonaws.com"
+                 + force_destroy               = false
+                   hosted_zone_id              = "XXXXXXXXXXXXX"
+                   id                          = "best-bucket-in-the-world"
+                   object_lock_enabled         = false
+                   region                      = "us-east-1"
+                   request_payer               = "BucketOwner"
+                 ~ tags                        = {
+                     - "foo" = "bar" -> null
+                   }
+                 ~ tags_all                    = {
+                     - "foo" = "bar"
+                   } -> (known after apply)
+
+                   grant {
+                       id          = "XXXXXXXXXXXXX"
+                       permissions = [
+                           "FULL_CONTROL",
+                       ]
+                       type        = "CanonicalUser"
+                   }
+
+                   server_side_encryption_configuration {
+                       rule {
+                           bucket_key_enabled = true
+
+                           apply_server_side_encryption_by_default {
+                               sse_algorithm = "AES256"
+                           }
+                       }
+                   }
+
+                   versioning {
+                       enabled    = true
+                       mfa_delete = false
+                   }
+               }
+
+           Plan: 1 to import, 0 to add, 1 to change, 0 to destroy.
+
+           ─────────────────────────────────────────────────────────────────────────────
+
+           Saved the plan to: plan
+
+           To perform exactly these actions, run the following command to apply:
+               terraform apply "plan"
+```
+
+### Importing without Configuration
+
+If you want to import a resource without specifying the configuration you can use the instance method `import` on the class of the resource you want to import. This method takes the scope as the first argument, the construct name within the CDKTF program as the second, resource ID as the third argument and the provider as an optional fourth. The provider is only required if you have multiple providers of the same type in your configuration.
+
+```typescript
+S3Bucket.import(this, "bucket", bucketId);
+```
+
+When running plan / apply you will get the information that one resource is going to be imported. Once you ran plan / apply, CDKTF CLI will return the CDKTF construct. You can use this in your CDKTF application after the the apply and therefore the import is done and replace the `S3Bucket.import` call.
```

Comment:
This needs some updating since `apply` will fail if the config is not yet present. I suspect the convert output has to be put into the config first (with an importFrom as specified above) and then an apply is needed.